--- htdig-3.1.4/CONFIG.in.unconfig	Thu Feb  4 21:47:22 1999
+++ htdig-3.1.4/CONFIG.in	Mon Nov 29 22:18:43 1999
@@ -23,19 +23,19 @@
 #
 # This specifies the root of the directory tree to be used by ht://Dig
 #
-DEST=                  $(prefix)
+DEST=                  @localstatedir@
 
 #
 # BIN_DIR
 # Set this macro to where you want the binaries to be installed.
 #
-BIN_DIR=               $(exec_prefix)/bin
+BIN_DIR=               @bindir@
 
 #
 # CONFIG_DIR
 # This is the directory that contains ht://Dig configuration files
 #
-CONFIG_DIR=            $(DEST)/conf
+CONFIG_DIR=            @sysconfdir@
 
 #
 # COMMON_DIR
